memory/vatest.py

```diff
@@ -44,7 +44,10 @@ def setUp(self):
         self.hsizes = [1024, 2]
         page_chunker = memory.meminfo.Hugepagesize.m
         if distro.detect().arch in ['ppc64', 'ppc64le']:
-            if 'power8' in cpu.get_cpu_arch():
+            mmu_detect = genio.read_file(
+                '/proc/cpuinfo').strip().splitlines()[-1]
+            # Check for "Radix" as this MMU will be explicit in POWER
+            if 'Radix' not in mmu_detect:
```

Comment:
older distros `cpuinfo` would not have `MMU` details, you might have to handle if this test to be run there aswell.

Reply:
Okay, let me try to handle.

Reply:
@harish-24 looks like we need this below commit, I hope all distros support Radix will have it.
```
commit 3a4c26011da3ed8df1f4f93af82bd5f2b420751f
Author: Aneesh Kumar K.V <aneesh.kumar@linux.vnet.ibm.com>
Date:   Tue Mar 21 22:59:55 2017 +0530

    powerpc/mm: Add translation mode information in /proc/cpuinfo
    
    With this we have on powernv and pseries /proc/cpuinfo reporting
    
    timebase        : 512000000
    platform        : PowerNV
    model           : 8247-22L
    machine         : PowerNV 8247-22L
    firmware        : OPAL
    MMU             : Hash
    
    Reviewed-by: Paul Mackerras <paulus@ozlabs.org>
    Signed-off-by: Aneesh Kumar K.V <aneesh.kumar@linux.vnet.ibm.com>
    Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
```